### DISCUSSION TITLE
Possible to add custom range on outpaint?

### DISCUSSION THREAD

**Discussion by Skitzum** [lllyasviel/Fooocus#2123] [Ideas]
It would be pretty cool to specify how much of each zone should be outpainted, instead of a static range. Ideal would be a quick preview with a slider that shows how much new area will be set to outpaint, a simpler option may be to outpaint small, medium, and large in the specified direction.

Even if the picture gets blurred it can still provide some background that can be further refined with inpaint.

**Comment by mashb1t** [comment]
This is a fixed value with 30%, which you can change/adjust on your installation. Larger values might lead to distorted images (most likely).

See code: https://github.com/lllyasviel/Fooocus/blob/main/modules/async_worker.py#L545-L562
